py-polars/polars/expr/expr.py

```diff
@@ -4265,6 +4265,12 @@ def map_elements(
             This method is much slower than the native expressions API.
             Only use it if you cannot implement your logic otherwise.
 
+            For mapping elements of columns, consider:
+            `pl.col("colname").something()`.
```

Comment:
thanks for your PR - not sure about using `.something` to be honest, I think this risks increasing confusion

Reply:
You are welcome to suggest another placeholder (I am stuck here).

Reply:
Perhaps, `.something()` → `.native_method()`?

Reply:
Renamed to `.native_method()`.

Reply:
Let us take the specific example of `.sqrt()`:

> For concreteness, suppose that the function is: `x ↦ sqrt(x)`.
> 
> For mapping elements of columns, consider:
> `pl.col("colname").sqrt()`.
> 
> For mapping elements of inner lists, consider:
> `pl.col("colname").list.eval(pl.element().sqrt())`.